### PR TITLE
Change LocationHours to a stateless component and show raw hours string

### PR DESCRIPTION
**Pull request by kelsonic**
## Description
**Issue:** https://github.com/department-of-veterans-affairs/va.gov-team/issues/504

**Current Behavior:** Nothing is rendered under location hours occasionally.

**Desired Behavior:** Always show the location hours and use `N/A`/etc. when the hours info is missing.

## Testing done
Locally.

## Screenshots
image

image

## Acceptance criteria
- [x] The frontend doesn't log an error for `Sunrise - Sunset`.
- [x] The frontend displays all hours.

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/facility-locator/components/LocationHours.jsx
Comment (ncksllvn) [inline src/applications/facility-locator/components/LocationHours.jsx]: Do we need to add a condition for when `hoursInfo` is falsy?

File: src/applications/facility-locator/components/LocationHours.jsx
Comment (ncksllvn) [inline src/applications/facility-locator/components/LocationHours.jsx]: I _think_ `vet_center` is a constant somewhere in the FL app directory, to help get rid of that magic string.

**Review by ncksllvn — COMMENTED** [review]
Looks like there are some unit tests that needs fixing, and may provide a bit more insight into what the other formats the hours may look like.

image

We may also need to provide a few sample screenshots in the issue to fly this by team members before merging to make sure they're in agreement on the approach.

File: src/applications/facility-locator/components/LocationHours.jsx
Comment (kelsonic) [reply src/applications/facility-locator/components/LocationHours.jsx]: That's a great question. Currently if `hoursInfo` is undefined, we show something like:

```
Sunday: N/A
Monday: N/A
Tuesday: N/A
Wednesday: N/A
...
```

Should we show nothing instead? I think this might be more of a design-related question -- do you perhaps know someone we can tag to get their input on this? :)

**Comment by kelsonic** [comment]
>Looks like there are some unit tests that needs fixing, and may provide a bit more insight into what the other formats the hours may look like.

Most of those unit tests are moot now since I made it a functional component so I'll go ahead and omit them.

I wonder if we should still format the hours when we can according to the unit tests and, if the hours are really malformed or not what we're expecting, then we just render the raw string instead of rendering nothing as we did previously.

**Comment by kelsonic** [comment]
@ncksllvn Updated and ready for review again 👍

**Comment by kelsonic** [comment]
Adding some screenshots of how it currently looks now:

image

image

image

image

image

**Review by ncksllvn — APPROVED** [review]
LGTM! Great work! If only we could just render the raw string 😭

**Review by U-DON — APPROVED** [review]

**Pull request merged**
